src/equations/2d/linear_scalar_advection.jl

```diff
@@ -48,36 +57,36 @@ end
 
 function initial_conditions_sin_sin(x, t, equation::LinearScalarAdvectionEquation2D)
   # Store translated coordinate for easy use of exact solution
-  x_trans = x - equation.advectionvelocity * t
+  x_trans = x_trans_periodic(x - equation.advectionvelocity * t)
```

Comment:
Here it is not necessary, since the IC (initial condition) is already periodic. Thus to reduce computational time, I suggest to revert it back to the original implementation.
```suggestion
  x_trans = x - equation.advectionvelocity * t
```